recipes/cas-offinder/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: cas-offinder
+  version: 2.4
+
+source:
+  url: https://github.com/snugel/cas-offinder/archive/5a5d24922005db5ab47dc0aceb0572a00b8f0e27.tar.gz
+  sha256: 7946039383fb826618959b6c729c8322355db59e2303ebaf1f2883f0365c103c
+
+requirements:
+  build:
+    - cmake # [unix]
```

Comment:
You can drop the ` # [unix]` selectors -- Bioconda only builds for Unix systems.

Reply:
Very nice. Thank you so much for the suggestion.